### PR TITLE
Update TLS parsing for multiple integrations

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Adds additional parsing for multiple `tls.*` fields.  

- Adds JA3/JA3S parsing & fixed not valid/before date fields in Zeek SSL log
- Adds JA3/JA3S parsing for Carbon Black EDR
- Add JA3/JA3S hash values to `related.hash` in Zeek SSL, Carbon Black, Fireeye NX, Suricata, Network Traffic integrations

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/carbonblack_edr/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -795,6 +795,23 @@ processors:
...
     ignore_missing: true
 
+- convert:
+    field: json.protocol
+    target_field: network.iana_number
+    type: string
+    ignore_missing: true
+- lowercase:
+    field: network.iana_number
+    ignore_missing: true
```
Comment (efd6) [inline packages/carbonblack_edr/data_stream/log/elasticsearch/ingest_pipeline/default.yml:805]: The IANA number should be a number, so is this necessary?

Comment (legoguy1000) [reply packages/carbonblack_edr/data_stream/log/elasticsearch/ingest_pipeline/default.yml:805]: ya, this can be removed. IDK why i added this

**Comment by efd6** [comment]
network_traffic needs ecs.yml updated to include `related.hash`.

**Comment by efd6** [comment]
Sorry, the version has changed under your feet.

File: packages/network_traffic/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # newer versions go on top
+- version: "1.3.0"
+    - description: Add JA3/JA3S to `related.hash`
```
Comment (efd6) [inline packages/network_traffic/changelog.yml:3]:
```suggestion
- version: "1.3.0"
  changes:
    - description: Add JA3/JA3S to `related.hash`
```

**Comment by legoguy1000** [comment]
@efd6 10th times the charm :(

**Review by efd6 — APPROVED** [review]
LGTM (thanks for hanging in there).

**Comment by legoguy1000** [comment]
@efd6 Can we merge?

**Pull request merged**